contrib/babelfishpg_common/sql/upgrades/babelfishpg_common--3.8.0--4.0.0.sql

```diff
@@ -0,0 +1,2 @@
+-- complain if script is sourced in psql, rather than via ALTER EXTENSION
```

Comment:
why is this needed now?

Reply:
As we need to put this to 3_X and in 3_X we will have 3.7 -> 3.8  then in 4_X we should have 3.8->4.0.